public/redux/reducers/__tests__/anomalyResults.test.ts

```diff
@@ -44,16 +44,19 @@ describe('elasticsearch reducer actions', () => {
         requesting: false,
         total: response.totalAnomalies,
         anomalies: response.results,
-        "featureData": undefined,
+        featureData: undefined,
       });
-      expect(httpMockedClient.get).toHaveBeenCalledWith(
+      expect(
+        httpMockedClient.get
+      ).toHaveBeenCalledWith(
         `..${AD_NODE_API.DETECTOR}/${tempDetectorId}/results`,
-        { params: queryParams }
+        { query: JSON.stringify(queryParams) }
```

Comment:
Good catch, in fact the reason the test was written this way was because the reducer is `stringify`ing the query, which isn't necessary. I've removed that altogether and kept as an object, and updated the tests. Confirmed it's still able to pull results correctly.

Reply:
Confirmed no other places are `stringify`ing a query. I think this was left over from debugging the original request parsing issues.

Reply:
Oh, got it. I didn't aware of the reducer. Thanks for testing it!